doc/bootstrap.rst

```diff
@@ -98,32 +112,32 @@ either as a simple IP address or as a CIDR network name.
 
 To deploy additional monitors,::
 
-  ceph orchestrator mon update *<new-num-monitors>* *<host1:network1> [<host1:network2>...]*
+  [monitor 1] # ceph orchestrator mon update *<new-num-monitors>* *<host1:network1> [<host1:network2>...]*
```

Comment:
Actually it doesn't matter where you issue this command. can be any client with an admin keyring.

Reply:
I get that, but this is the first step in updating the Getting Started Guide so that it explains how to install a Cluster with three monitors in it. I'd like to keep this heuristic device in here as a first-day-of-school walkthrough.

I'll add a note in the future, if you prefer, explaining that this is just a first walkthrough and that the full truth will be revealed later.

Reply:
Ok for me! I'll leave it to you then.

Reply:
Roger that, sebastian-philipp, and thanks.